### DISCUSSION TITLE
Speed Variation - Fast = Failure, under 3mbps = success

### DISCUSSION THREAD

**Discussion by birdybirdybird** [aclap-dev/video-downloadhelper#1346] [Help and Q&A]
### Your question

I've noticed that if the speed ramps up past 3mbps when I'm downloading, it will fail at about 15-20% in to the file. If it slowly builds to high 1 - low 2 mbps, it is successful. If I'm on the same site, it might be one day it works, one day it fails... and another day it isn't successful at all. Any ideas? 

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "http_media_download_strategy": "inbrowser",
    "used_history_button": true,
    "download_directory": "/Users/minitest/Downloads/",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/Users/minitest/Documents"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 18,
    "successfull_dl": 14,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I might have some ideas if you gave me at least one URL of a web page where a video resides that exhibits the behavior you are describing.  At least one.  A few would be acceptable.  Simply the name of a web site is not acceptable.  Don't tell me, "Anything on site x will do."  Give me concrete examples.

I must say I've never encountered the situation you describe.  Occasionally, I'll see the opposite.  If the download starts out slow & never speeds up, it's doomed.  But if it starts out fast & does speed up, it succeeds . . . usually.  There are exceptions, but more often than not, it completes successfully.

So you have to give me examples or I can't help you.
